### ISSUE TITLE
Missing instance types

### ISSUE THREAD

**Issue by chrislo2004** [jantman/awslimitchecker#429]
Hi Jantman,
  Before hand, thanks for making a great tool to be used, I really appreciate it!

  Ok, so there is an error on unknown instance type 't3a.xlarge'; not counting, hope you can fix this!

version: awslimitchecker 7.1.0

Thanks & Regards

**Comment by jantman** [comment]
@chrislo2004 apologies for the delayed response to this.

awslimitchecker has a hard-coded list of EC2 instance types that it supports. It appears that none of the ``t3a`` instances have been added yet. I'll make sure to get this into the next release, though I can't commit to a specific timeline.

**Comment by jantman** [comment]
It appears that there are currently 42 instance types missing:

```
c5.12xlarge
c5.24xlarge
c5.metal
c5n.metal
i3en.12xlarge
i3en.24xlarge
i3en.2xlarge
i3en.3xlarge
i3en.6xlarge
i3en.large
i3en.metal
i3en.xlarge
m5.16xlarge
m5.8xlarge
m5.metal
m5a.16xlarge
m5a.8xlarge
m5ad.12xlarge
m5ad.24xlarge
m5ad.2xlarge
m5ad.4xlarge
m5ad.large
m5ad.xlarge
m5d.16xlarge
m5d.8xlarge
m5d.metal
r5a.16xlarge
r5a.8xlarge
r5ad.12xlarge
r5ad.24xlarge
r5ad.2xlarge
r5ad.4xlarge
r5ad.large
r5ad.xlarge
t3a.2xlarge
t3a.large
t3a.medium
t3a.micro
t3a.nano
t3a.small
t3a.xlarge
z1d.metal
```

**Comment by jantman** [comment]
Many apologies for the delay in implementing this... it's occasionally difficult to balance life and maintaining an open source project in my spare time. I plan on trying to get a fix for this out in the next week.

**Comment by chrislo2004** [comment]
No worries but I think AWS changed to vhost instead of instance as of
today, so now am finding a way for getting the vhost for the threshold
limit alert health check

**Comment by jantman** [comment]
@chrislo2004 I'm working on implementing changes for that right now, see #432. There will be a new release out sometime this week that includes updates for the new vCPU-based limits.

Since AWS can't ever make anything simple, the new vCPU limits aren't being rolled out in the China or US GovCloud regions, so the instance type updates will need to happen as well, in order to support users in those regions.

**Comment by jantman** [comment]
This will be present in the next release, though it will only be helpful for people using the China and GovCloud regions.

**Comment by jantman** [comment]
This has been fixed in [8.0.0](https://github.com/jantman/awslimitchecker/releases/tag/8.0.0) which is now live on [PyPI](https://pypi.org/project/awslimitchecker/) and [Docker Hub](https://hub.docker.com/r/jantman/awslimitchecker/tags). Apologies for any delays in getting this released.
